### PR TITLE
Feat/Update views

### LINKED ISSUE #205: Update Income Graphs/View
Update the income graphs on base of the design in Zeplin

### LINKED ISSUE #211: Update Expense Graphs/View
Update hook, graphs and view based on of design

### LINKED ISSUE #212: Update Delegate Graphs/View
Update hook, graphs and view based on of design

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #205 
- Resolve #211
- Resolve #212 

### Steps to test

1. Run Project
2. Enter to Income, Expense and Delegate
imagen
imagen
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by JAMares** [comment]
This PR is missing some details, but i how is so big (sorry for that), i make it now

File: webapp/src/utils/new-format-objects.js [hunk 1]
```diff
@@ -105,37 +57,144 @@ export const newDataFormatPercentByElection = (
...
+    [`EOS_UN${uppercaseCategory}_PERCENT`]:
+      Number(data[`eos_un${category}`]) * 100,
+    [`USD_${uppercaseCategory}_PERCENT`]: Number(data[`usd_${category}`]) * 100,
+    [`USD_UN${uppercaseCategory}_PERCENT`]:
+      Number(data[`usd_un${category}`]) * 100
   }))
 }
 
-export const newDataFormatExpensesAcrossElections = transactionsList =>
+// Income
```
Comment (AngeloCG97) [inline webapp/src/utils/new-format-objects.js:69]: We need this comment

File: webapp/src/utils/new-format-objects.js [hunk 2]
```diff
@@ -105,37 +57,144 @@ export const newDataFormatPercentByElection = (
...
+
+export const newDataFormatTotalByCategory = totalByCategory =>
+  totalByCategory.map(data => ({
+    name: data.category ? data.category : `Election ${data.election + 1}`,
+    EOS_CATEGORIZED: Number(data.amount),
+    USD_CATEGORIZED: Number(data.usd_total),
+    color: generateColor()
+  }))
+
+// Delegate
```
Comment (AngeloCG97) [inline webapp/src/utils/new-format-objects.js:175]: We need this comment

**Review by AngeloCG97 — COMMENTED** [review]
Please view comments

File: webapp/src/utils/new-format-objects.js (hunk 1 above)
Comment (JAMares) [reply webapp/src/utils/new-format-objects.js:69]: Is for know by view although can add the name view and delete the comment

File: webapp/src/utils/new-format-objects.js [hunk 3]
```diff
@@ -105,37 +57,144 @@ export const newDataFormatPercentByElection = (
...
+  transactionsList.map(data => ({
+    name: data.recipient,
+    EOS_CLAIMED: Number(data.eos_claimed),
+    USD_CLAIMED: Number(data.usd_claimed),
+    EOS_UNCLAIMED: Number(data.eos_unclaimed),
+    USD_UNCLAIMED: Number(data.usd_unclaimed),
+    color: generateColor()
+  }))
+
+// Expense
```
[the remark this answers is not in the document]
Comment (JAMares) [reply webapp/src/utils/new-format-objects.js:102]: Is for know by view although can add the name view and delete the comment

File: webapp/src/utils/new-format-objects.js (hunk 2 above)
Comment (JAMares) [reply webapp/src/utils/new-format-objects.js:175]: Is for know by view although can add the name view and delete the comment

File: webapp/src/components/BarChartGeneralReport/index.js [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+  const handleBarDownload = useCallback(async () => {
+    const png = await getBarPng()
+
+    if (png) {
+      FileSaver.saveAs(png, 'bar-chart.png')
+    }
+  }, [getBarPng])
+
+  useEffect(() => {
+    typeData === 'income'
```
Comment (tetogomez) [inline webapp/src/components/BarChartGeneralReport/index.js:190]: Avoid nested ternary if declaration

**Review by tetogomez — CHANGES_REQUESTED** [review]

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**